testing/test_repo.py

```diff
@@ -4,19 +4,7 @@
 from pipsi import Repo, find_scripts
 
 
-@pytest.fixture(params=['normal', 'MixedCase'])
-def mix(request):
-    return request.param
-
-
-@pytest.fixture
-def bin(tmpdir, mix):
-    return tmpdir.ensure(mix, 'bin', dir=1)
-
-
-@pytest.fixture
-def home(tmpdir, mix):
-    return tmpdir.ensure(mix, 'venvs', dir=1)
+pytestmark = pytest.mark.usefixtures("home", "bin", "mix")
```

Comment:
I've never used this marker in my tests and always just used the fixtures. @RonnyPfannschmidt if this is not required, is it idiomatic?